preprocessing/tabular_data/preprocessing_tabular_data.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<p>Did you mean to say "future split" here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>